src/app/core/models/github/cache-manager/comments-cache-manager.model.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * A model that is used to manage the etags of multiple list of issues' comments paginated by pages.
+ * Example of a single list of comments: https://developer.github.com/v3/issues/comments/#list-comments-in-a-repository
+ */
+import { GithubComment } from '../github-comment.model';
+import { GithubResponse } from '../github-response.model';
+
+export class CommentsCacheManager {
```

Comment:
May I clarify why we need this cache manager? Doesn't the IssuesCacheManager suffice?

Reply:
Thanks. I have removed it, along with anything related to comment api since we are using graphql which fetches comments together with issue.